pyTANSPEC/reduce_specTANSPEC.py

```diff
@@ -1188,8 +1193,15 @@ def CombDith_FlatCorr_subrout(PC,method="median"):
             Flats2Comb=set(Flats2Comb)  #Making a set to remove duplicates
             #Write all these flat names to a file.
             imgflatlistfname=os.path.join(PC.RAWDATADIR,PC.OUTDIR,night,OutCombimg[:-5]+'.flatlist')
-            with open(imgflatlistfname,'w') as imgflatlistFILE :
-                imgflatlistFILE.write('\n'.join([PC.RAWDATADIR+'/'+night+'/'+fla for fla in Flats2Comb])+'\n')
+            flatlist = open(imgflatlistfname, 'w')
+            flatlist.close()
```

Comment:
Why is this file opened and closed? Is it to erase the contents?

Reply:
This command is redundant, actually. After few lines, the same file is opened and adding things. This opening and closing is not needed.